vm/qemu/qemu.go

```diff
@@ -165,6 +165,7 @@ var linuxCmdline = []string{
 	"vsyscall=native",
 	"net.ifnames=0",
 	"biosdevname=0",
+	"dummy_hcd.num=8",
```

Comment:
This should also go into dashboard/config/upstream-*.cmdline

Reply:
I think it's better to do it in another PR together with enabling USB fuzzing related configs when we start deploying this on syzbot.